content/docs/1.42/deployment.md

```diff
@@ -498,7 +498,7 @@ docker run \
 #### Shards and Replicas for Elasticsearch indices
 
 Shards and replicas are some configuration values to take special attention to, because this is decided upon
-index creation. [This article](https://qbox.io/blog/optimizing-elasticsearch-how-many-shards-per-index) goes into
+index creation. [This article](https://aws.amazon.com/blogs/database/choosing-the-right-number-of-shards-for-your-large-scale-amazon-dynamodb-table/) goes into
```

Comment:
this doesn't make sense, the new link is to DynamoDB, not Elasticsearch. I assume the official Elastic website might have some docs about shards.

Reply:
Also, I would assume this link is present in `deployment.md` for many versions.

Reply:
I have made the necessary changes. 

Thank You Very Much For The Code Review!